numbast/src/numbast/tools/static_binding_generator.py

```diff
@@ -101,6 +101,8 @@ class Config:
      module_callbacks : dict[str, str]
          Dictionary containing setup and teardown callbacks for the module.
          Expected keys: "setup", "teardown". Each value is a string callback function.
+     skip_prefix : str | None
+         Do not generate bindings for any functions that start with this prefix
```

Comment:
Documentation should state the behavior when `skip_prefix` is unspecified. If `None`, we should be including all APIs.